Gopkg.toml

```diff
@@ -61,6 +61,14 @@ ignored = [
   name = "github.com/golang/glog"
   source = "github.com/istio/glog"
 
+[[override]]
+  name = "k8s.io/helm"
+  version = "2.11.0"
+
+[[override]]
```

Comment:
would be nice to standardize on one log framework.  uber's zap is pretty straightforward to use.

Reply:
istio also uses glog, and if I cannot use logrus, then I prefer to use glog..

Reply:
We have our own logging library under /pkg/log. We should use that.

Reply:
In fact, I 

> @ozevren @ostromart when I remove my local vendor I see this failure in CI, appreciate some help to understand why it cannot import these libraries even though they do exist in global vendor folder:
> 
> ```
> tools/tiller-hack/tiller-hack.go:29:7:error: could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (maligned)
> tools/tiller-hack/tiller-hack.go:29:7:error: error return value not checked (could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of:) (errcheck)
> tools/tiller-hack/tiller-hack.go:29:2:error: cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (vet)
> tools/tiller-hack/tiller-hack.go:29:2:error: cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (vetshadow)
> tools/tiller-hack/tiller-hack.go:29:7:error: unused struct field could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (structcheck)
> tools/tiller-hack/tiller-hack.go:29:7:error: could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (unconvert)
> tools/tiller-hack/tiller-hack.go:29:7:error: could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (interfacer)
> tools/tiller-hack/tiller-hack.go:29:7:error: unused variable or constant could not import k8s.io/helm/pkg/proto/hapi/release (cannot find package "k8s.io/helm/pkg/proto/hapi/release" in any of: (varcheck)
> ```

Have you tried dep update? It is possible that the vendored code is pruned.